### DISCUSSION TITLE
Ran tests, test result is ok but 0 tests run

### DISCUSSION THREAD

**Discussion by Abelaby** [Cyfrin/foundry-full-course-cu#323] [Q&A] [answered]
Im a bit confused, I was writing tests for the checkUpKeep() function in Lesson 9 and when I ran the test I got the following output.

```terminal
forge test --mt checkUpkeepReturnsFalseIfItHasNobalance
[⠢] Compiling...
No files changed, compilation skipped

Test result: ok. 0 passed; 0 failed; 0 skipped; finished in 6.70ms
Ran 1 test suites: 0 tests passed, 0 failed, 0 skipped (0 total tests)
```

This is my checkUpkeepReturnsFalseIfItHasNobalance() function:

```solidity
function checkUpkeepReturnsFalseIfItHasNobalance() public {
        vm.warp(block.timestamp + interval + 1);
        vm.roll(block.number + 1);

        (bool upKeepNeeded, ) = raffle.checkUpKeep("");
        assert(!upKeepNeeded);
    }
```

I dont understand why there are 0 tests passed?

**Comment by ghost** [comment, marked as answer]
According to the docs functions need to be prefixed with the word `test` to be run as test cases. 

Try with this name: `testCheckUpkeepReturnsFalseIfItHasNobalance`.

**Reply by Abelaby** [reply]
It worked!!!!!!!

Thanks a lot mate.
Guess we learn something new every time
